### DISCUSSION TITLE
Неправильно работает mod Simple Image Upload, если установлен Optimus

### DISCUSSION THREAD

**Discussion by realdigger** [dragomano/Optimus#42] [General]
Если установлены Simple Image Upload и Optumus, загрузчик картинок цепляется к полю "Описание темы [SEO]" вместо окна редактора. 

![issue](https://github.com/dragomano/Optimus/assets/1187218/5ccf7835-3ce1-4af9-8d5a-914bfeefc13a)

**Comment by dragomano** [comment]
Как я понял, этот плагин цепляется ко всем textarea на странице. В файле `imgbb.js` можно дополнить список-исключений, указав параметр name `optimus_description`:

```js
var imgbb_skip_textarea = new RegExp(
  'recaptcha|username_list|search|recipients|signature|optimus_description',
  'i'
);
```
